infrastructure/prometheus/prometheus.yml

```diff
@@ -26,4 +26,4 @@ scrape_configs:
 
```

Comment:
I think 'prometheus:19090' needs to be changed to 'prometheus:9090' (with 19090 I get 'connection refused' in Prometheus).

Reply:
fixed